x-pack/plugins/ml/public/application/app.tsx

```diff
@@ -75,7 +75,15 @@ const App: FC<AppProps> = ({ coreStart, deps, appMountParams, isServerless, mlFe
   const pageDeps: PageDependencies = {
     history: appMountParams.history,
     setHeaderActionMenu: appMountParams.setHeaderActionMenu,
-    setBreadcrumbs: coreStart.chrome!.setBreadcrumbs,
+    setBreadcrumbs: (breadcrumbs) => {
+      coreStart.chrome!.setBreadcrumbs(breadcrumbs);
+      if (deps.serverless) {
```

Comment:
the `serverless` plugin dependency is unnecessary, an `isServerless` flag is already being passed into this function.

Reply:
Ignore this, I noticed serverless.setBreadcrumbs is used